### ISSUE TITLE
`git pw series` needs `update` feature

### ISSUE THREAD

**Issue by andy-shev** [getpatchwork/git-pw#26]
Update series is usually what would be nice to have.

**Comment by stephenfin** [comment]
Currently the only thing that can be updated is series names, which simply aren't exposed. What would be more useful is updating series state but that's blocked on the server side by https://github.com/getpatchwork/patchwork/issues/157. One that's resolved, we should definitely do this.

**Comment by borkmann** [comment]
Would love to have that feature, so I could set the entire series simply to accepted. Any progress on this? Thx

**Comment by stephenfin** [comment]
It should be in 3.0, though that's a good six months to a year out, unfortunately. At present, we're stuck with a DB issue that no one has yet figured out (though someone is working on it)
